### ISSUE TITLE
Host a demo website with the notification plugin 

### ISSUE THREAD

**Issue by johnhooks** [WordPress/wp-feature-notifications#192]
## What problem does this address?

Having a demo available will make it easier for people to experience the features of the plugin and provide feedback.

## What is your proposed solution?

@Sephsekla do you know of some standard way we host this some where in WordPress development land? Then we can add staging deployment to the CI workflow.

**Comment by johnhooks** [comment]
@erkiyo and I had a conversation about trying to use the [WordPress Playground](https://developer.wordpress.org/playground/) to create a easy to share demo of the plugin. The big bonus being able to auto login as a user without haven't to pass someone credentials and leaving a demo site open to abuse.

**Comment by johnhooks** [comment]
@erikyo I'm looking into how to implement this, I believe you have plans for Storybook to be output to the docs folder so it can be hosted though GitHub pages. Can we put it into a subdirectory of `docs` and create a website for the project? The website could then contain the documentation and a plugin demo with WordPress Playground.

**Comment by erikyo** [comment]
we can provide a target folder to storybook, for the moment I setup the `./docs/`

https://github.com/WordPress/wp-feature-notifications/blob/97f37599cbe8ad907f83fa936bb12a0b204168cb/storybook/package.json#L7

but we can choose a different target or, even better, create a page to host the playground. What do you think @johnhooks, would not be cool?

**Comment by johnhooks** [comment]
Yes @erikyo this is exactly what I meant. It would be cool and help people discover the project.

**Comment by erikyo** [comment]
Ok cool! as we get merged the storybook updates we can work to this one, I made a different package.json from the repo and that should make our life even easier
